plugins/fortinet_fortiswitchmanager.rb

```diff
@@ -0,0 +1,17 @@
+Plugin.define do
+  name "fortinet fortiswitchmanager"
+  authors [
+    "Ostorlab",
+  ]
+  version "0.1"
+  description "FortiSwitch Manager enables network administrators to cut through the complexities of non-FortiGate-managed FortiSwitch deployments."
+  website "https://docs.fortinet.com/product/fortiswitch-manager"
+
+  matches [
+    {
+      :search => "body",
+      :regexp => /FortiSwitchManager/,
```

Comment:
this one also, if you can find something more special, it's very easy to find those words in the body of other websites.